components/shared/AppSidebar/AppSidebar.tsx

```diff
@@ -92,204 +92,209 @@ export const AppSideBar = ({ workspaceId, hideSidebar, sidebarCollapsed }: AppSi
   return (
     // TODO: get rid of the z-index. There is grid content like the avatars and paged data text that bleed through the sidebar atm.
     <div
-      className={`fixed left-0 w-72 bg-white shadow-lg transform  transition-transform duration-300 ease-in-out border-r ${
-        sidebarCollapsed ? "" : "-translate-x-full"
+      aria-hidden={sidebarCollapsed}
+      className={`fixed left-0 w-72 bg-white shadow-lg transform transition-transform duration-300 ease-in-out border-r ${
+        sidebarCollapsed ? "-translate-x-full" : ""
       } bg-white flex flex-col gap-8 justify-between max-w-xs  border-r border-slate-200 z-50`}
       style={{
         "--top-nav-height": "3.3rem",
         top: "var(--top-nav-height)",
         height: "calc(100dvh - var(--top-nav-height))",
       }}
     >
-      <nav aria-label="sidebar navigation" className="grid gap-4 mt-4 pr-4 pl-2">
-        <div className="flex gap-2">
-          <label className="workspace-drop-down flex flex-col w-full gap-2 ml-2">
-            <span className="sr-only">Workspace</span>
-            <SingleSelect
-              options={[
-                { label: "Create new workspace...", value: "new" },
-                ...workspaces.map(({ id, name }) => ({
-                  label: name,
-                  value: id,
-                })),
-              ]}
-              position="popper"
-              value={workspaceId ?? "new"}
-              placeholder="Select a workspace"
-              onValueChange={(value) => {
-                if (value === "new") {
-                  router.push("/workspaces/new");
-                  return;
-                }
+      {sidebarCollapsed ? null : (
```

Comment:
Now we only render the links if the side bar is not collapsed. I had considered other options like `visibility: hidden` or `display: none`, but they're not supported for animations, and broke the close animiation.